src/test/java/ru/training/at/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,99 @@
+package ru.training.at.hw2.ex1;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class Exercise1 {
+    private WebDriver webDriver;
+
+    @BeforeTest
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver",
+                "C:\\Users\\Iuliia_Gorshkova\\Downloads\\chromedriver_win32\\chromedriver.exe");
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
+    }
+
+    @AfterTest
+    public void tearDown() {
+        if (webDriver != null) {
+            webDriver.close();
+        }
+    }
+
+    @Test
+    public void exerciseOneTest() {
+
+        //  1. Open test site by URL
```

Comment:
Let's think about inheritance. You could dedicate some common logic from both classes in additional parent class, make it as test e.g. Also the same could be done with some common expected values - they could be dedicated as well. That changes have you to rethink Before and After methods in order to make project works. If you need in clues type me in Teams about that.